smex/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please mention in the commit message why we need this tool as @mmaka1  said.

Reply:
Added